### PR TITLE
fix(alias): allow multiple check aliases

### PR DESCRIPTION
**Pull request by MrCloudSec**
### Context

When a Check Alias applies to multiple checks, Prowler only executes the first one.

### Description

Execute all the applicable check to the specific alias.

### License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: prowler/lib/check/checks_loader.py [hunk 1]
```diff
@@ -110,15 +112,20 @@ def update_checks_to_execute_with_aliases(
...
-        ):
-            # Remove input check name and add the real one
-            checks_to_execute.remove(input_check)
-            checks_to_execute.add(check_aliases[input_check])
-            print(
-                f"\nUsing alias {Fore.YELLOW}{input_check}{Style.RESET_ALL} for check {Fore.YELLOW}{check_aliases[input_check]}{Style.RESET_ALL}...\n"
-            )
-    return checks_to_execute
+    try:
+        new_checks_to_execute = checks_to_execute.copy()
```
Comment (jfagoagas) [inline prowler/lib/check/checks_loader.py:116]: Why do you need this copy? If you use a `set` you are not going to have duplicates in that object.

**Review by jfagoagas — CHANGES_REQUESTED** [review]

File: prowler/lib/check/checks_loader.py (hunk 1 above)
Comment (MrCloudSec) [reply prowler/lib/check/checks_loader.py:116]: Because I only want to modify the copied set, not the original one since it is being iterated.

Comment (MrCloudSec) [reply prowler/lib/check/checks_loader.py:116]: If the original set is modified while iterating through it, it will throw an error.

Comment (jfagoagas) [reply prowler/lib/check/checks_loader.py:116]: Why would that happen? We were removing checks from the `checks_to_execute`.

**Review by jfagoagas — APPROVED** [review]
Great one! Thanks @sergargar 🚀

**Pull request merged**